files/statsd_mapping.conf

```diff
@@ -0,0 +1,12 @@
+test.dispatcher.*.*.*
```

Comment:
It's the default supplied by the [`statsd_exporter`](https://github.com/prometheus/statsd_exporter)